### PR TITLE
chore: upgrade to latest Keycloak chart

### PR DESCRIPTION
**Pull request by aledegano**
While the current version of Keycloak is not explicitly affected by Log4shell, it is however safer to ship a more recent version.

/deploy

### CODE REVIEW

**Comment by rokroskar** [comment]
did you test to see what happens when you upgrade from the current version to this new one? Any db migrations to be aware of?

**Comment by aledegano** [comment]
> did you test to see what happens when you upgrade from the current version to this new one? Any db migrations to be aware of?

No, I didn't do any tests, I thought it was a trivial upgrade, but as the deployment shows it isn't really. I'll close this PR to avoid confusion. I believe @ableuler will pick this up when his pipeline clears up.

**Comment by rokroskar** [comment]
What was the problem with the deployment?

**Comment by rokroskar** [comment]
It did deploy, actually: https://github.com/SwissDataScienceCenter/renku/runs/4725939662?check_suite_focus=true

> I believe @ableuler will pick this up when his pipeline clears up.

😆 hope dies last

**Comment by aledegano** [comment]
> It did deploy, actually: https://github.com/SwissDataScienceCenter/renku/runs/4725939662?check_suite_focus=true
> 
> > I believe @ableuler will pick this up when his pipeline clears up.
> 
> laughing hope dies last

it deploys, but the last time I've checked all the tests were failing to authenticate...

**Comment by rokroskar** [comment]
@aledegano @ableuler I tested this in rok.dev.renku.ch by first installing the current `master` and then upgrading with this PR. Seems to work normally so I think the PR is gtg.

**Comment by aledegano** [comment]
Wow, great, thanks for picking this up!

File: Pipfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ name = "pypi"
 [packages]
 "urllib3" = ">=1.24.2"
 "jinja2" = ">=2.10.1"
-chartpress = "==0.7.0"
+chartpress = "==1.0.0"
```
Comment (ableuler) [inline Pipfile:9]: Wasn't there a reason for pinning this to 0.7.0 @aledegano ?

Comment (rokroskar) [reply Pipfile:9]: CI is using 1.0... I think that was part of the reason for the weird build discrepancy we were looking at last week @ableuler

Comment (aledegano) [reply Pipfile:9]: IIRC the 0.7.0 was the latest version that still supported having the chartpress file in a different folder than that it expected.
However this was ~1 year ago and things might have changed(?)

Comment (aledegano) [reply Pipfile:9]: https://github.com/jupyterhub/chartpress/issues/118

Comment (rokroskar) [reply Pipfile:9]: afaict all of our chart building actions work fine with this change. But maybe I'm not hitting all the places that could break?

Comment (aledegano) [reply Pipfile:9]: :shrug:

Comment (ableuler) [reply Pipfile:9]: alright then, let's merge this and remind ourselves that this might be a problem if pipelines suddenly started to fail :)

**Pull request merged**